sources/System/FileSystem/FileSystem.h

```diff
@@ -15,6 +15,8 @@
 
 #define SAMPLE_LIB_PATH "/samplelib"
 
+#define PAGED_PAGE_SIZE 18
```

Comment:
Why here and not in picoTraclerFileSystem.h? a matter of scope for use in PagedImportSampleDialog.cpp?

Reply:
Its because currently the UI code only references the abstractions of the `FileSystem.h` and never ` picoTraclerFileSystem.h` which is the "hidden implementation" of the interfaces defined in  `FileSystem.h`.
This does kind of leaks that abstraction by exposing the page size concept, but I thought it was the least of 2 evils, but happy to do it differently if you have a suggestion?